gkeys/gkeys/actions.py

```diff
@@ -393,9 +403,10 @@ def speccheck(self, args):
                 results = self.gpg.speccheck(gkey.keydir, key)
                 for g in results:
                     pub_pass = {}
+                    key_print = ''
                     for key in results[g]:
                         self.output('', key.pretty_print())
-
+                        key_print += '\n\n' + key.pretty_print()
```

Comment:
why do you need the key_print variable?  You only ever assign to it once and use it once.